### PR TITLE
add ability to remove from cart - cart.js

### PR DESCRIPTION
**Pull request by ashesmsmith**
all other changes from npm run format

### CODE REVIEW

File: src/js/cart.js [hunk 1]
```diff
@@ -45,6 +52,20 @@ function cartItemTemplate(item) {
...
 
+function deleteFromCart(event) {
+  const itemId = event.target.dataset.id;
+
+  if (!itemId) return;
+
+  const cartItems = getLocalStorage(cartLSKey) || [];
+
+  const updatedCart = cartItems.filter((item) => item.Id !== itemId);
+  localStorage.setItem(cartLSKey, JSON.stringify(updatedCart));
```
Comment (Abuddabi) [inline src/js/cart.js]: @ashesmsmith I would suggest to use here setLocalStorage function from utils.mjs

Comment (ashesmsmith) [reply src/js/cart.js]: I made the change suggested. Do I need to delete this pull request and start a new one?

Comment (Abuddabi) [reply src/js/cart.js]: no, you made a new commit to the current branch, it's enough.
great work!

**Review by Abuddabi — APPROVED** [review]
great work!

**Pull request merged**